pootle/apps/pootle_store/migrations/0027_unit_created_by_squashed_0055_fill_unit_source_data.py

```diff
@@ -0,0 +1,437 @@
+                ('commented_on', models.DateTimeField(db_index=True, null=True)),
+                ('reviewed_on', models.DateTimeField(db_index=True, null=True)),
+                ('commented_by', models.ForeignKey(null=True, on_delete=models.SET(pootle.core.user.get_system_user), related_name='units_commented', to=settings.AUTH_USER_MODEL)),
+                ('reviewed_by', models.ForeignKey(null=True, on_delete=models.SET(pootle.core.user.get_system_user), related_name='units_reviewed', to=settings.AUTH_USER_MODEL)),
+                ('submitted_by', models.ForeignKey(null=True, on_delete=models.SET(pootle.core.user.get_system_user), related_name='units_submitted', to=settings.AUTH_USER_MODEL)),
+                ('unit', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, related_name='change', to='pootle_store.Unit')),
+            ],
+            options={
+                'abstract': False,
+                'db_table': 'pootle_store_unit_change',
+            },
+        ),
+        migrations.RunPython(convert_unit_source_change),
+        migrations.RunPython(
+            remove_fields,
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='source_hash',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='source_length',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='source_wordcount',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='commented_by',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='commented_on',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='reviewed_by',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='reviewed_on',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='submitted_by',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='submitted_on',
+        ),
+        migrations.AlterField(
+            model_name='unitchange',
+            name='commented_by',
+            field=models.ForeignKey(null=True, on_delete=models.SET(pootle.core.user.get_system_user), related_name='commented', to=settings.AUTH_USER_MODEL),
+        ),
+        migrations.AlterField(
+            model_name='unitchange',
+            name='reviewed_by',
+            field=models.ForeignKey(null=True, on_delete=models.SET(pootle.core.user.get_system_user), related_name='reviewed', to=settings.AUTH_USER_MODEL),
+        ),
+        migrations.AlterField(
+            model_name='unitchange',
+            name='submitted_by',
+            field=models.ForeignKey(null=True, on_delete=models.SET(pootle.core.user.get_system_user), related_name='submitted', to=settings.AUTH_USER_MODEL),
+        ),
```

Comment:
they cant - they set the related fields that were previously used by unit attributes